.exp-insertions.yml

```diff
@@ -141,15 +142,18 @@ jobs:
   steps:
   - checkout: self
 
-  - checkout: CloudBuildConfig
-
   - powershell: |
-      $json = (Get-Content "$(Build.SourcesDirectory)/CloudBuildConfig/$(_MSBuildConfigFilePath)" -Raw) | ConvertFrom-Json 
-      $MSBuildDropPath = $json.Tools.MSBuild.Locations
+      $url = "https://dev.azure.com/cloudbuild/CloudBuild/_apis/git/repositories/CloudBuildConfig/items?versionDescriptor.version=main&path=$(_MSBuildConfigFilePath)&api-version=5.0"
```

Comment:
Nit: Let's define this link as a variable, similar how `_MSBuildConfigFilePath` is defined. Or maybe even instead of this variable, since it is the only place where we use it.

Reply:
Updated